### PR TITLE
Updated start project section on homepage

### PR DESCRIPTION
**Pull request by lucgeurts**
Changed to the new design for project start on the homepage. Also cleaned out non-used code (e.g. carousel and related css code).

### CODE REVIEW

File: CollAction/Frontend/app/project/Carousel.tsx [hunk 1]
```diff
@@ -1,212 +0,0 @@
-import * as React from "react";
```
Comment (timstokman) [inline CollAction/Frontend/app/project/Carousel.tsx]: We don't need the carousel anymore at all? @ljwboschker Does that seem right?

Comment (ljwboschker) [reply CollAction/Frontend/app/project/Carousel.tsx]: No, this was the last place it was used. Files can be removed; no use in keeping old stuff around.

File: CollAction/Frontend/app/project/styles/SimpleProjectCreate.scss [hunk 2]
```diff
@@ -183,38 +183,4 @@
...
-    i {
-      float: right;
-    }
-
-    span {
-      margin-right: 10px;
-    }
-
-  }
-}
```
Comment (ljwboschker) [inline CollAction/Frontend/app/project/styles/SimpleProjectCreate.scss]: I think this whole file can be removed; it was for the previous start account page.

Comment (lucgeurts) [reply CollAction/Frontend/app/project/styles/SimpleProjectCreate.scss]: I've delete this file and tested it 👍

**Comment by ljwboschker** [comment]
Approved.

**Pull request merged**